incubator/perfsonar-testpoint-demo/images/supervisord.conf

```
@@ -0,0 +1,47 @@
+[supervisord]
+nodaemon=true
```

```suggestion
nodaemon=false
```
Let's daemonize Supervisor. This will prevent its logs from clogging up `slate instance logs`.

---

Thanks for your comments on this!
This setting is coming from upstream docker image and I am using it as is. The container keeps crashing if I change it to false. This change would need more researching and testing which, I think, is not necessary for this chart.

---

As per our subsequent discussions with the team, this wouldn't change.